packages/terra-card/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "terra-card",
+  "main": "lib/Card.js",
+  "version": "1.0.0",
+  "description": "{Card is a basic container with some base styling to help seperate elements with different content}",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-card",
+    "Card",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^1.0.0"
```

Comment:
I think you missed @emilyrohrbough 's previous comment
https://github.com/cerner/terra-core/pull/721/files#r132976584